### ISSUE TITLE
Checkbox has no visible :focus style

### ISSUE THREAD

**Issue by kyleoliveiro** [auth0/cosmos#871]
A `:focus` style is essential for keyboard users trying to use a `Checkbox` component.

**Comment by kyleoliveiro** [comment]
This issue also applies to the following components. Not sure whether to create a separate issue for each, so I'll list them here for now:

**Components lacking visual `:focus` style**
- Radio
- Switch

**Comment by landitus** [comment]
Seems we can tackle all of them in one pass! We can first enable the default browser style and then check whether we want to tweak the style all throughout the system.

**Comment by DCzajkowski** [comment]
I am working on it! I should have a PR ready any time today 😉

**Comment by francocorreasosa** [comment]
Resolved in #960
